src/webgpu/util/floating_point.ts

```diff
@@ -3758,7 +3758,7 @@ class F32Traits extends FPTraits {
       'unpack2x16snormInterval only accepts values on the bounds of u32'
     );
     const op = (n: number): FPInterval => {
-      return this.maxInterval(this.divisionInterval(n, 32767), -1);
+      return this.maxInterval(this.ulpInterval(n / 32767, 3), -1);
```

Comment:
Since n is an integer in a finite range, and the divisor is a fixed integer, it's safe to do the division in JS and then take the ULP interval.

I think the all unpack unorm/snorm functions can be simplified this way.